### DISCUSSION TITLE
End Capture/Stream capabilities when doing a Live Stream or Motion Capture

### DISCUSSION THREAD

**Discussion by AM-Novus** [gopro/labs#239] [Feature Requests]
I have use cases where I would love to have control over being able to end Motion Capture on after 'X' Seconds after starting motion capture, and also be able to end a live stream after 'X' seconds of starting stream. In other words is it possible to implement a command as following:

1.) !SM2+5E (Start motion capture when finding anything more than a senstivity of 2 and end the motion capture after 5 seconds.

2.) Similarly, is it possible to implement something like "!GM+600E" which implies Start a live stream and end it after 600 Seconds.

Thanks

**Comment by dnewman-gpsw** [comment]
Return to the other thread and you can try both of this ideas, although the syntax will be different.
